source/solvers/fluid_dynamics_matrix_free_operators.cc

```diff
@@ -1323,7 +1476,7 @@ NavierStokesStabilizedOperator<dim, number>::do_cell_integral_local(
             {
               // (-ν∆δu + (u·∇)δu + (δu·∇)u)·τ∇q
```

Comment:
Could you update those comments or add the corresponding one for the new operator?